### DISCUSSION TITLE
[JetBrains] Autocomplete is broken after the update to 1.1.0

### DISCUSSION THREAD

**Discussion by jakubtobiasz** [community/community#8043] [Copilot Conversations] [answered]
Hi!
Today I've updated Copilot for JetBrains IDE (PhpStorm in my case) to 1.1.0. After this update I've noticed the autocompletion is broken. Randomly after pressing TAB it deletes the everything before suggestion and puts only the suggested part. Below I'm adding a GIF to visualize the problem.
![broken_copilot](https://user-images.githubusercontent.com/80641364/143090831-83129cee-e16d-4e1f-8dc8-f3fa2c9dbd87.gif)

**Comment by jansorg** [comment, marked as answer]
@jakubtobiasz Thanks for reporting and the GIF! We'll fix this shortly.

**Reply by jansorg** [reply]
1.1.1 should fix this issue. Please let us know when you're still seeing it!

**Reply by jakubtobiasz** [reply]
I've been coding on 1.1.1 for a while and it seems everything is ok.
